app/gui/StreamSegue.qml

```diff
@@ -72,11 +73,15 @@ Item {
             // Run the streaming session to completion
             session.exec(Screen.virtualX, Screen.virtualY)
```

Comment:
This is going to be slightly problematic for multi-monitor configurations, since the user won't have the ability to position the UI on their preferred display. Maybe we should add an option to specify the launch monitor?

I don't think this needs to be done before merging though.

Reply:
Yeah, that sound good idea to me.

Reply:
So, I tested this a bit. 

It seems that Qt doesn't offer a simple way to identify a display, like some integer 1, 2, ... etc. There's `QGuiApplication::screens()` which returns list of `QScreen`, but at least on my Win10 machine with two displays connected, both `QScreen`s return empty strings for `model()` and `serialNumber()`. `name()` returns `"\\\\.\\DISPLAY2"` and `"\\\\.\\DISPLAY1"`.
On Ubuntu running inside Virtualbox with one display, I get `"VGA-1"` as name. Model and serialNumber are empty there as well.

Perhaps this could work so that, there would a new command line action for listing displays, e,g,:
```
> moonlight.exe list-displays
\\.\DISPLAY2
\\.\DISPLAY1
```
And then you could pass the display name as an option to stream action:
```
> moonlight.exe stream MyAwesomePC "XCOM 2" --display \\.\DISPLAY1
```